jbpm-wb-forms/jbpm-wb-forms-client/src/main/java/org/jbpm/workbench/forms/client/display/views/display/EmbeddedFormDisplayerViewImpl.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.forms.client.display.views.display;
+
+import javax.annotation.PostConstruct;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.dom.client.DivElement;
+import com.google.gwt.dom.client.Document;
+import com.google.gwt.dom.client.SpanElement;
+import com.google.gwt.dom.client.Style;
+import com.google.gwt.user.client.ui.Composite;
+import org.gwtbootstrap3.client.ui.gwt.FlowPanel;
+import org.jboss.errai.ui.shared.api.annotations.DataField;
+import org.jboss.errai.ui.shared.api.annotations.Templated;
+import org.jbpm.workbench.forms.client.display.GenericFormDisplayer;
+
+@Templated
+public class EmbeddedFormDisplayerViewImpl extends Composite implements EmbeddedFormDisplayerView {
+
+    @DataField
```

Comment:
If using `@Dependent`, you could initialize these fields using `@Inject` instead?

Reply:
@pefernan fair enough. In that case is best to keep `GWT.create` so that GWTMockitoRunner can mock it.

Reply:
@cristianonicolai restored GWT.create. Now let's wait for jenkins hehe